core/src/main/java/com/adobe/aem/commons/assetshare/components/actions/share/impl/EmailShareServiceImpl.java

```diff
@@ -105,6 +109,12 @@ public boolean accepts(final SlingHttpServletRequest request) {
 
     @Override
     public final void share(final SlingHttpServletRequest request, final SlingHttpServletResponse response, final ValueMap shareParameters) throws ShareException {
+    	
+    		/** Work around for regression issue introduced in AEM 6.4 **/
+        SlingBindings bindings = new SlingBindings();
+        bindings.setSling( new ScriptHelper(bundleContext, null, request, response));
```

Comment:
@godanny86 it might be useful to add a more explicit comment here stating that the second argument is intentionally null since we don't have a `SlingScript` to pass in. Otherwise, you might forget later why that is null.

Reply:
Good call, thanks @justinedelson.